pika/connection.py

```diff
@@ -672,9 +684,21 @@ def __init__(self,
         # Define our callback dictionary
         self.callbacks = callback.CallbackManager()
 
+        # Attributes that will be properly initialized by _init_connection_state
+        # and/or during connection handshake.
```

Comment:
REVIEWME Initialize attributes in constructor that were previously initialized only elsewhere.